html/inc/email.inc

```diff
@@ -63,11 +64,18 @@ function send_email($user, $subject, $body, $body_html=null, $email_addr=null) {
         } else if (defined('EMAIL_FROM')) {
             $headers = "From: ". EMAIL_FROM;
         }
-        if ($email_addr) {
-            return mail($email_addr, $subject, $body, $headers);
-        } else {
-            return mail($user->email_addr, $subject, $body, $headers);
+        if (!$email_addr) {
+            $email_addr = $user->email_addr;
+        }
+        if ($body_html) {
+            $body = "<html>\n"; 
+            $body .= '<body style="font-family:Verdana, Verdana, Geneva, sans-serif; font-size:12px;">\n'; 
```

Comment:
Wouldn't it be better to leave off the style tag on the body element?  If a project is using HTML emails they are probably trying to replicate the styles they are using on their website and things like font and size may be something they want to control.